helm/values.yaml

```
@@ -278,12 +278,14 @@ defaults: &defaults
 
   ingress:
     enabled: true
-    annotations: {
-      "nginx.ingress.kubernetes.io/rewrite-target": "$1",
-      "nginx.org/server-snippet": "proxy_ssl_verify off;"
-    }
+    annotations:
+      nginx.ingress.kubernetes.io/rewrite-target: '$1',
+      nginx.org/server-snippet: 'proxy_ssl_verify off;'
```

```suggestion
    annotations: {}
```
Annotations should default empty because they'll be different for different ingress implementations (i.e. Traefik, instead of nginx)